packages/test-generator/lib/generators/NodeTestGenerator.ts

```diff
@@ -74,7 +74,7 @@ export class NodeTestGenerator extends TestGenerator {
       (schemas: StudioSchema[]) => new ReactIndexStudioTemplateRenderer(schemas, this.renderConfig),
     );
     this.utilsRendererFactory = new StudioTemplateRendererFactory(
-      (utils: Util[]) => new ReactUtilsStudioTemplateRenderer(utils, this.renderConfig),
+      (utils: UtilTemplateType[]) => new ReactUtilsStudioTemplateRenderer(utils, this.renderConfig),
```

Comment:
Did you get a TS error here? I had it when I changed to union type.

Reply:
by having the class accept the utiltemplate type it will expect an array of those types instead of an array of strings
```ts
export type UtilTemplateType = 'validation' | 'formatter';

export class ReactUtilsStudioTemplateRenderer extends StudioTemplateRenderer<
  string,
  UtilTemplateType[],
  ReactOutputManager,
  {
    comp
```

Reply:
For some reason when I did this, I got a TS error here that I could not understand. That's why I changed UtilTemplateType to string, until I could figure it out.